npm-package/index.d.ts

```diff
@@ -156,6 +156,6 @@ export interface EnhancerOptions {
   };
 }
 
-export function composeWithDevTools(...funcs: Function[]): GenericStoreEnhancer;
+export function composeWithDevTools(...funcs: Function[]): StoreEnhancer;
 export function composeWithDevTools(options: EnhancerOptions): typeof compose;
-export function devToolsEnhancer(options: EnhancerOptions): GenericStoreEnhancer;
+export function devToolsEnhancer(options: EnhancerOptions): StoreEnhancer;
```

Comment:
Could you, please, mark `options` as an optional argument? That is, append a `?`. It is, in fact, optional, as per the docs and the source code:

https://github.com/zalmoxisus/redux-devtools-extension/blob/0caa713ea629a2cb58d3ad78cb6c8675ab2d95bf/src/browser/extension/inject/pageScript.js#L31

Asking you, @codeaid, instead of making a PR, because it'll likely take several more months to merge it.

Reply:
and....

Reply:
@FDiskas I gave up waiting for maintainer merging this so forked and patched. As temporary solution https://www.npmjs.com/package/redux-devtools-extension-sol

Reply:
Sorry, @jeremejevs, completely missed your comment! I guess it's irrelevant now anyway seeing how this PR got closed.

Reply:
No worries, @codeaid, this is no showstopper :slightly_smiling_face: